app/views/components/show.html.erb

```diff
@@ -1,39 +1,40 @@
-<h1><%= @component.name %></h1>
+<h1 class="govuk-heading-xl"><%= @component.name %></h1>
+
+<h2 class="govuk-heading-l"><%= @component.component_type %></h2>
 
-<table class="govuk-table">
-  <h3 class="govuk-heading-m">Certificates to upload</h3>
-  <thead class="govuk-table__head">
-    <tr class="govuk-table__row">
-      <th class="govuk-table__header" scope="col">Name</th>
-      <th class="govuk-table__header" scope="col">Type</th>
-    </tr>
-  </thead>
-  <tbody class="govuk-table__body">
-      <tr class="govuk-table__row">
-        <td class="govuk-table__cell" scope="row"><%= @component.name %></td>
-        <td class="govuk-table__cell"><%= @component.component_type %></td>
-      </tr>
-  </tbody>
 
 </table>
 <%= link_to 'Upload', new_component_certificate_path(@component), class: 'govuk-button' %>
 
-<% if @component_certificates.present? %>
-  <table class="govuk-table">
-    <h3 class="govuk-heading-m">Certificates currently in use</h3>
-    <thead class="govuk-table__head">
+<h3 class="govuk-heading-m">Signing Certificates (Enabled)</h3>
+<% @certificates =  @component.enabled_signing_certificates %>
+<%= render "certificates/certificates_table" %>
+
+<h3 class="govuk-heading-m">Signing Certificates (Disabled)</h3>
+<% @certificates =  @component.disabled_signing_certificates %>
+<%= render "certificates/certificates_table" %>
+
+
+<% if @component.certificates.present? %>
+    <table class="govuk-table">
+      <h3 class="govuk-heading-m">All certificates for this component</h3>
+      <thead class="govuk-table__head">
       <tr class="govuk-table__row">
+        <th class="govuk-table__header" scope="col">ID</th>
         <th class="govuk-table__header" scope="col">Usage</th>
+        <th class="govuk-table__header" scope="col">Enabled</th>
         <th class="govuk-table__header" scope="col">Last updated</th>
       </tr>
-    </thead>
-    <tbody class="govuk-table__body">
-      <% @component_certificates.each do |certificate| %>
-        <tr class="govuk-table__row">
-          <td class="govuk-table__cell" scope="row"><%= certificate.usage %></td>
-          <td class="govuk-table__cell"><%= format_date_time(certificate.updated_at) %></td>
-        </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+      <% @component.certificates.each do |certificate| %>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell" scope="row"><%= certificate.id %></td>
+            <td class="govuk-table__cell" scope="row"><%= certificate.usage %></td>
+            <td class="govuk-table__cell" scope="row"><%= certificate.enabled %></td>
+            <td class="govuk-table__cell"><%= format_date_time(certificate.updated_at) %></td>
+          </tr>
       <% end %>
-    </tbody>
-  </table>
-<% end %>
+      </tbody>
+    </table>
```

Comment:
There are a couple of inconsistencies between this table and the ones above it now:

- the columns are in a slightly different order (last updated fourth here, first above)
- the `h3` tag is still inside the `table` tag here (outside above, to avoid inclusion in the template I assume?)

Reply:
This table is a slightly modified version of the existing one, that is only remaining until we run the equivalent ticket for encryption certs. The h3 tag shouldn't be in the table. Pretty sure that's not even valid html 😧